### DISCUSSION TITLE
SRT stream and tweaking

### DISCUSSION THREAD

**Discussion by gre2gor** [bluenviron/mediamtx#3768] [Questions]
### Question

Hi, how can I set maxbw, latency, buffer for a SRT connection? It seems now I can only add SRT, but I see maxbw is always at "-1", is there a way to set it?

**Comment by aler9** [comment]
SRT parameters can be set in the usual way, by using parameters in the SRT URL:

```
ffmpeg -re -f lavfi -i testsrc=size=1920x1080:rate=30 -pix_fmt yuv420p -c:v libx265 -g 60 -preset medium -x265-params crf=26:bframes=4 -f mpegts 'srt://localhost:8890?streamid=publish:test&pkt_size=1316&maxbw=100&latency=200&sndbuf=300'
```

The problem of `maxbw` is that it's [currently unimplemented by the underlying library](https://github.com/datarhei/gosrt/blob/e535ac940024074cbd1b06b217bbd505b6730cd6/README.md?plain=1#L168). Open an issue or contribute to datarhei/gosrt if you want it to be implemented.
